Zero-Gravity-Store-React/src/App.js

```diff
@@ -10,15 +10,17 @@ import { Login } from "./Pages/AuthWidget/Login/Login";
 import { SignUp } from "./Pages/AuthWidget/SignUp/SignUp";
 import { Logout } from "./Pages/AuthWidget/Logout/Logout";
 import Mockman from "mockman-js";
+import SingleProductPage from "./Pages/SingleProductPage/SingleProductPage";
```

Comment:
there is nothing wrong with this approach but you can consider making an indej.js file in pages folder where you would export All pages and whenever you need it, you can import from there all at once instead of importing each page from each file.